src/crypto/ocb_openssl.cc

```diff
@@ -71,6 +88,7 @@ int ae_init(ae_ctx *ctx, const void *key_param, int key_len, int nonce_len,
 int ae_encrypt(ae_ctx *ctx, const void *nonce_ptr, const void *pt_ptr,
 	       int pt_len, const void *ad_ptr, int ad_len, void *ct_ptr,
 	       void *tag, int final) {
+  assert(final == AE_FINALIZE);
```

Comment:
Other unsupported bits like `nonce_len` have done a runtime check with `return AE_NOT_SUPPORTED`. Would that be better here? Ditto below.